extensions/wikia/GlobalFooter/GlobalFooterController.class.php

```diff
@@ -80,6 +86,8 @@ private function getGlobalFooterLinks() {
 			}
 		}
 
+		$wgMemc->set( $memcKey, $parsedLinks, self::MEMC_EXPIRY );
```

Comment:
I believe if message key exists we should display its value despite what it contains and if it contains anything. As you said that might be an error but on the other hand that may be intended. Messages are controlled by us so I believe we can assume if key exists then we should show whatever value it contains.